.circleci/config.yml

```diff
@@ -13,7 +13,10 @@ jobs:
         type: boolean
         default: true
     docker:
-      - image: circleci/golang:<< parameters.go_version >>
+      - image: cimg/go:<< parameters.go_version >>
+    environment:
+      # Override Go 1.18 security deprecations.
+      GODEBUG: "x509sha1=1,tls10default=1"
```

Comment:
```
      GODEBUG: "x509sha1=1,tls10default=1"
```
So this does the trick for certificate failures of the tests! Is there documentation somewhere for these parameters?

Reply:
It's in the 1.18 release notes.